### ISSUE TITLE
How to remove confirmation when selecting a command from the run list.

### ISSUE THREAD

**Issue by ioogithub** [OrkoHunter/keep#74]
I am trying out keep. I want the ability to type "keep run" and select a number from the list:

keep run 3

I do not want to have to press Y/n every time, this is quite annoying.

I see you can type a command "keep run command -n" and not get the conformation. How can I remove the confirmation when selecting a command from the run list?

For keep to be useful it needs to save time over doing a reverse search.

**Comment by OrkoHunter** [comment]
Hi @ioogithub ! `-n` was shipped in the latest release. Does it solve your problem i.e. remove confirmation?

Do you think the default should be to not have confirmation?

I like the confirmation bit, it is explicit and helps me see what I am doing. But I think there should be a config to disallow if a user wants it. :)

**Comment by ioogithub** [comment]
> Hi @ioogithub ! `-n` was shipped in the latest release. Does it solve your problem i.e. remove confirmation?

Does the -n flag work if you are selecting a task from the numbered list?

`keep run`
list displayed..
`3`
Are you sure [Y/n]

I can see that the -n command can be used if you execute your command like this:

keep run "my command" -n

but does it work with a list? I tried and couldn't get it working.

The issue is that when you run keep run and select the item from the list the "pause" in the workflow is already built into the moment where you have to stop and read the list and pick the item you want.

I understand how a confirmation can be useful if you are typing out the whole command with the alias all at once because there is no pause here.

When using the keep run command then typing a number the pause is a second pause and really gets in the way.

I tried to save my commands with a -n but it ignores it. Am I missing something or is -n functionality missing from when the user pick the command from the list?

**Comment by OrkoHunter** [comment]
Good point! The `-n` is newly added and can be improved. It should stop confirmation when selecting from list as well. +1 from me, if you want to create a new issue, rename this one, or maybe work on it. :)

Thanks anyways.
